nck/config.py

```diff
@@ -20,6 +20,8 @@
 import os
 
 LEVEL = logging.INFO
+if "LOGGING_LEVEL" in os.environ:
+    LEVEL = os.environ["LOGGING_LEVEL"]
```

Comment:
One way to handle default values for `dict` is to use the `.get()` method. What do you think about this?
```suggestion
LEVEL = os.environ.get("LOGGING_LEVEL", logging.INFO)
```

Reply:
yes you're right it is shorter and clearer.